### PR TITLE
fix: fixed the background height to suit smaller screens

### PR DESCRIPTION
**Pull request by darshansimha**
Screenshot 2023-10-23 at 2 46 07 PM

### CODE REVIEW

File: ui/src/components/pages/Login/index.tsx [hunk 1]
```diff
@@ -292,6 +298,40 @@ export function Login() {
...
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          marginTop: "5rem",
+        }}
+      >
+        <Box
+          sx={{
+            color: "#fff",
+            fontSize: "32px",
```
Comment (bbehnke) [inline ui/src/components/pages/Login/index.tsx:314]: rem please.

File: ui/src/components/pages/Login/index.tsx [hunk 2]
```diff
@@ -292,6 +298,40 @@ export function Login() {
...
+            display: "flex",
+            flexDirection: "row",
+          }}
+        >
+          Unlock the power of
+        </Box>
+        <Box
+          sx={{
+            color: "#fff",
+            fontSize: "32px",
```
Comment (bbehnke) [inline ui/src/components/pages/Login/index.tsx:324]: +1

**Review by bbehnke — CHANGES_REQUESTED** [review]

File: ui/src/components/pages/Login/index.tsx (hunk 1 above)
Comment (darshansimha) [reply ui/src/components/pages/Login/index.tsx]: updated

**Review by bbehnke — APPROVED** [review]
LGTM

**Pull request merged**